util/access_token.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	pullSecretKey     = ".dockerconfigjson"
+	pullSecretKey     = ".dockerconfigjson" //#nosec G101 -- This is a false positive
```

Comment:
I am wondering if we should fix the "confusing" name of the const.

And it is interesting there the next line is not being caught?